modules/Calendar/Cal.js

```diff
@@ -151,11 +151,11 @@ CAL.remove_edit_dialog();var cal_loaded=true;$($.fullCalendar).ready(function(){
 +'<div class="qtip-title-buttons">'
 +'</div>';var body=SUGAR.language.translate('app_strings','LBL_LOADING_PAGE');if($('#cal_module').val()!="Home"){element.qtip({content:{title:{text:title,button:true,},text:body,},events:{render:function(event,api){$.ajax(url).done(function(data){eval(data);var divCaption="#qtip-"+api.id+"-title";var divBody="#qtip-"+api.id+"-content";if(data.caption!=""){$(divCaption).html(result.caption);}
-api.set('content.text',result.body);}).fail(function(){$(divBody).html(SUGAR.language.translate('app_strings','LBL_EMAIL_ERROR_GENERAL_TITLE'));}).always(function(){});}},position:{my:'bottom left',at:'top left'},show:{solo:true},hide:{event:false},style:{width:224,padding:5,color:'black',textAlign:'left',border:{width:1,radius:3},tip:'bottomLeft',classes:{tooltip:'ui-widget',tip:'ui-widget',title:'ui-widget-header',content:'ui-widget-content'}}});}},}).ready(function(){$(window).resize();});if($('#calendar_title_'+user_id).length==0){var calendar=$("#calendar"+user_id+" > .fc-view-container");var calendarTitle="<div class='monthCalBody'><h5 class='calSharedUser' id='calendar_title_"+user_id+"'></h5></div><div id='calendar"+user_id+"'></div>";$(calendarTitle).prependTo(calendar);$.ajax({url:"index.php?module=Calendar&action=getUser&record="+user_id,}).done(function(data){data=jQuery.parseJSON(data);$("#calendar_title_"+user_id).html(data.full_name);});}}});
\ No newline at end of file
+api.set('content.text',result.body);}).fail(function(){$(divBody).html(SUGAR.language.translate('app_strings','LBL_EMAIL_ERROR_GENERAL_TITLE'));}).always(function(){});}},position:{my:'bottom left',at:'top left'},show:{solo:true},hide:{event:false},style:{width:224,padding:5,color:'black',textAlign:'left',border:{width:1,radius:3},tip:'bottomLeft',classes:{tooltip:'ui-widget',tip:'ui-widget',title:'ui-widget-header',content:'ui-widget-content'}}});}},}).ready(function(){$(window).resize();});if($('#calendar_title_'+user_id).length==0){var calendar=$("#calendar"+user_id+" > .fc-view-container");var calendarTitle="<div class='monthCalBody'><h5 class='calSharedUser' id='calendar_title_"+user_id+"'></h5></div><div id='calendar"+user_id+"'></div>";$(calendarTitle).prependTo(calendar);$.ajax({url:"index.php?module=Calendar&action=getUser&record="+user_id,}).done(function(data){data=jQuery.parseJSON(data);$("#calendar_title_"+user_id).html(data.full_name);});}}});
```

Comment:
Please also update jssource/src_files/modules/Calendar/Cal.js and include in your pull request. I think it is around line 1070